daemon/config/config_windows.go

```diff
@@ -15,7 +15,7 @@ type BridgeConfig struct {
 // to the docker daemon when you launch it with say: `dockerd -e windows`
 type Config struct {
 	CommonConfig
-
+	NetworkConfig
```

Comment:
Still not sure why this is being added on `windows` if you say it isn't used, but at least now the struct isn't duplicated.

Reply:
missed this one. will remove it.